blueprints/empress-blog/files/__base__/content/advanced-markdown.md

```diff
@@ -10,6 +10,14 @@ authors:
 date: Tue Jun 12 2018 17:50:55 GMT+0100 (IST)
 tags:
   - getting-started
+meta:
```

Comment:
same comment as above regarding having default metadata in generator files. Also, just a minor thing: categories seems a little bit close to tags 🤔  do you have a better example to use? this is only a soft suggestion so if not then it's ok 👍

Reply:
Same answer as above.

Re: Categories, I'll think of something else.